README.md

```diff
@@ -434,6 +434,21 @@ To enable Prometheus, use the `--prometheus` flag. This will start an HTTP
 server on localhost with a `/metrics` endpoint. The Prometheus namespace may
 optionally be set with `--prometheus-namespace`.
 
+## Debug server
+
+The Proxy includes support for a debug server on localhost. By default, the
+debug server is not enabled. To enable the server, pass the `--debug` flag.
+This will start the server on localhost at port 9191. To change the port,
```

Comment:
Should we reuse the http handler for readiness checks and metrics?

Reply:
We'll make this 9091.

Reply:
We'll change this to `---admin-port`.